products/opensuse/main.pm

```diff
@@ -741,6 +741,29 @@ sub load_slenkins_tests {
     return 0;
 }
 
+sub load_security_tests {
```

Comment:
Don't have a lot of expertise in this area but brief grep of sle/main.pm give me that you mainly collect here tests which are split in SLE into several functions :
 - load_fips_tests_core
 - load_fips_tests_web 

As a person who invest a lot of time trying to merge extra_tests  in opensuse and sle I want to ask if there good reason to organize SLE FIPS tests in opensuse differently then in opensuse. I understand that some tests are not valid for opensuse but still methodology can be the same. Can you please take a look at https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_common.pm#L447 and think about how to apply this to opensuse+SLE FIPS tests ?

Reply:
I purposed to create a group to include all security related tests and make simple structure. But I agree with your suggestion to drop them in EXTRA, also to eliminate duplicated tests in both of group.

Reply:
sorry probably I confuse you , I didn't meant to ask you to add fips tests inside EXTRA test I just want to show them as a good example of how we should manage SLE+opensuse together. Currently extra tests already running too long ( we even had cases when test was failing because it reach 2 hours absolute limit and openQA just shut it down ) so better to keep fips tests as separate test suite but just organize code in same way as it done for EXTRA

Reply:
Ok, I see. I also considered it. Actually I did it in local, but sometimes hesitate which one is better :)
see following runs:
Tumbleweed Non-FIPS:
http://10.67.17.71/tests/42
http://10.67.17.71/tests/43
http://10.67.17.71/tests/44
http://10.67.17.71/tests/49
SLE FIPS:
http://10.67.17.71/tests/50
http://10.67.17.71/tests/52
http://10.67.17.71/tests/53
http://10.67.17.71/tests/54